### ISSUE TITLE
Training Epoch stuck at (1/2)

### ISSUE THREAD

**Issue by hugogo10123** [xuannianz/EfficientDet#228]
Hugode-MBP:EfficientDet hugo$ python3 train.py --no-snapshots --phi 0 --gpu 0 --random-transform --compute-val-loss --freeze-backbone --batch-size 32 --steps 4 pascal /Users/hugo/Downloads/VOCdevkit_2012_train/VOC2012
{'dataset_type': 'pascal', 'detect_quadrangle': False, 'detect_text': False, 'snapshot': None, 'freeze_backbone': True, 'freeze_bn': False, 'weighted_bifpn': False, 'batch_size': 32, 'phi': 0, 'gpu': '0', 'epochs': 10, 'steps': 4, 'snapshot_path': 'checkpoints/2020-12-24', 'tensorboard_dir': 'logs/2020-12-24', 'snapshots': False, 'evaluation': True, 'random_transform': True, 'compute_val_loss': True, 'multiprocessing': False, 'workers': 1, 'max_queue_size': 10, 'pascal_path': '/Users/hugo/Downloads/VOCdevkit_2012_train/VOC2012'}

2020-12-24 00:44:54.475679: I tensorflow/python/profiler/internal/profiler_wrapper.cc:87] Creating directory: logs/2020-12-24/train/plugins/profile/2020_12_24_00_44_54Dumped tool data for overview_page.pb to logs/2020-12-24/train/plugins/profile/2020_12_24_00_44_54/Hugode-MBP.overview_page.pb
Dumped tool data for input_pipeline.pb to logs/2020-12-24/train/plugins/profile/2020_12_24_00_44_54/Hugode-MBP.input_pipeline.pb
Dumped tool data for tensorflow_stats.pb to logs/2020-12-24/train/plugins/profile/2020_12_24_00_44_54/Hugode-MBP.tensorflow_stats.pb
Dumped tool data for kernel_stats.pb to logs/2020-12-24/train/plugins/profile/2020_12_24_00_44_54/Hugode-MBP.kernel_stats.pb

4/4 [==============================] - ETA: 0s - loss: 1.7616 - classification_loss: 1.2421 - regression_loss: 0.5194

======================================================================================

The training program can't go to Epoch (2/2), only keep at Epoch (1/2). My command is :
python3 train.py --no-snapshots --phi 0 --gpu 0 --random-transform --compute-val-loss --freeze-backbone --batch-size 32 --steps 4 pascal /Users/hugo/Downloads/VOCdevkit_2012_train/VOC2012
Could you help me how to fix it?
